src/components/Posts/ExpandedPostReplies.tsx

```diff
@@ -22,22 +22,23 @@ const ExpandedPostReplies = ({ post }: ExpandedPostRepliesProps) => {
           {
             params: {
               userId: user.userId,
-              postId: post.postId,
+              postId: postId,
             },
           }
         );
         dispatch(setPosts(resultReplies.data as Post[]));
+        console.log(resultReplies);
```

Comment:
think we just need this console.log removed

Reply:
Nice catch, thanks!